### DISCUSSION TITLE
Failed to connect to browser

### DISCUSSION THREAD

**Discussion by bastiH96** [Second-Hand-Friends/kleinanzeigen-bot#310] [Q&A]
Hello,

I'm a little confused how to setup the browser configuration part in the config.yaml file. When I run the program to publish some ads I always getting this error message:

<img width="830" alt="Screenshot 2024-06-02 at 06 23 08" src="https://github.com/Second-Hand-Friends/kleinanzeigen-bot/assets/67004154/a532e89c-3852-4f1c-b98c-b87cff3e84c6">

Any help would be appreciated.

**Comment by fabsGitHub** [comment]
Hello everyone,

I have the same problem, even if I use no_sandbox. 

<img width="1541" alt="Bildschirmfoto 2024-07-07 um 13 25 34" src="https://github.com/Second-Hand-Friends/kleinanzeigen-bot/assets/54660763/1be29990-5911-4805-9914-dd73ff6348f1">

any ideas?

**Reply by bastiH96** [reply]
I have still no solution and keep hoping for help

**Comment by fabsGitHub** [comment]
this solved my problem:

https://github.com/Second-Hand-Friends/kleinanzeigen-bot/issues/315#issuecomment-2184019359

**Reply by bastiH96** [reply]
Doesn't work. It caused a new error that asked me to remove the "remote-debugging-port"-line.

Edit: This seems to be a problem with macOs. I ran the bot with same configs on my windows maschine and it worked flawlessly!
